### ISSUE TITLE
3D Sound

### ISSUE THREAD

**Issue by vitvakatu** [three-rs/three#115]

**Comment by Rhuagh** [comment]
Rodio have support for 3d now I believe.

**Comment by vitvakatu** [comment]
@Rhuagh, yes, I know. The only reason it's not implemented yet - it requires solution of some architectural questions in the engine core
